backend/uasmartsignage/src/main/java/deti/uas/uasmartsignage/Services/FileService.java

```diff
@@ -26,9 +28,17 @@ public class FileService {
     
     private final FileRepository fileRepository;
 
+    private final LogsService logsService;
+
+    private static final String SOURCE = "FileService";
```

Comment:
I might be wrong but I think that you could use the following code to prevent this hardcoded string
`this.getClass().getSimpleName();`